fs-cache/src/cache.rs

```diff
@@ -0,0 +1,60 @@
+use crate::memory_limited_storage::MemoryLimitedStorage;
+use data_error::Result;
+use std::path::Path;
+
+pub struct Cache<K, V> {
+    storage: MemoryLimitedStorage<K, V>,
+}
```

Comment:
`Cache` is just a very thin wrapper around `MemoryLimitedStorage`, to the point where it doesn't really add any new functionality. 

Why do we need it? Are there plans to extend it in the future?

Reply:
Initially, I implemented a few functions specifically for `Cache`, but Kirill suggested that we currently only need the `get`/`set` API. Therefore, I think it's a good idea to remove the wrapper and directly expose this API.

Reply:
Removed it